app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -32,7 +32,11 @@ class MoveToHousehold extends React.Component {
   }
 
   handleChange(event) {
-    this.setState({ [event.target.id]: event.target.value, updateDisabled: false });
+    let updateDisabled = true;
+    if (event.target.id == 'hoh_selection') {
+      updateDisabled = !this.state.groupMembers.map(p => p.id?.toString()).includes(event.target.value?.split('SaraID: ')[1]?.split(' ')[0]);
```

Comment:
Digging into this a little bit, what if the split part of split('SaraID: ')[1] is null, and then we try to grab the second thing in there. I can't see that happening unless someone deleted what was in the datalist and typed gibberish, but could it happen?. I suppose it means the button is still disabled, but could cause some confusion when the JS stops executing because this line.

Reply:
![image](https://user-images.githubusercontent.com/3967953/86157505-7dd2f880-bad5-11ea-93ff-d3911f3ad0ac.png)

Reply:
Perfect!